rasa/nlu/training_data/formats/markdown.py

```diff
@@ -90,22 +88,28 @@ def _strip_comments(text: Text) -> Text:
         return re.sub(comment_regex, "", text)
 
     @staticmethod
-    def _create_section_regexes(section_names):
-        def make_regex(section_name):
+    def _create_section_regexes(section_names: List[Text]) -> Dict[Text, Pattern]:
+        def make_regex(section_name: Text) -> Pattern:
             return re.compile(r"##\s*{}:(.+)".format(section_name))
 
         return {sn: make_regex(sn) for sn in section_names}
 
-    def _find_section_header(self, line):
+    def _find_section_header(self, line: Text) -> Optional[Tuple[Text, Text]]:
         """Checks if the current line contains a section header
         and returns the section and the title."""
         for name, regex in self.section_regexes.items():
```

Comment:
do we even need this if we have 

```
match = re.search(r"##\s*(.+):(.+)", line)
if match is not None:
     return match.group(1), match.group(2)
```

?

Reply:
true, will change it.